testing/.eslintrc.js

```diff
@@ -19,5 +19,14 @@ module.exports = {
     'local/no-export-side-effect': 0,
     'local/no-module-exports': 0,
     'local/no-style-property-setting': 0,
+    'local/no-spread': 0,
```

Comment:
The spread operator is used by `TestConfig` + related code now that it needs to pass through the args of both `describes` and `describe`. The purpose of this rule is to prevent the need for unnecessary polyfills in runtime code, so it's safe to ignore in `testing/` since its contents aren't shipped with the runtime.